src/utils/__tests__/fields.test.js

```diff
@@ -19,6 +19,7 @@ import {
     FIELD_PARENT,
     FIELD_EXTENSION,
     FIELD_PERMISSIONS,
+    FIELD_PROMOTED_BY,
```

Comment:
same comment about grouping with the other `[...]_BY` fields

Reply:
This list is sorted alphabetically